components/classes/digital-asset.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/digital-asset",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Digital Asset",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Digital asset such as an image or video.",
+  "definitions": {
+    "digital-asset": {
+      "properties": {
+        "xdm:digitalAsset": {
+          "title": "Digital Asset",
+          "description": "Digital asset such as an image or video.",
+          "type": "object",
+          "properties": {
+            "xdm:version": {
+              "title": "Version",
+              "type": "string",
+              "description": "Asset version."
+            },
+            "xdm:perceptionId": {
```

Comment:
following XDM coding guidelines, please change the field name from `xdm:perceptionId` to `xdm:perceptionID`